site/content/en/docs/tasks/dev/integrate_a_custom_job.md

```diff
@@ -92,6 +93,18 @@ Here are completed external integrations you can learn from:
 Add your framework's GroupVersionKind to `.integrations.externalFrameworks` in [controller_manager_config.yaml](
 https://kueue.sigs.k8s.io/docs/installation/#install-a-custom-configured-released-version).
 
+### RBAC Augmentation
+
+Kueue will need permission to get, list, and watch instances of your CRD.
```

Comment:
I'm confused. I don't think Kueue needs this permissions, only your external binary.

Unless you are forking Kueue but, in that case, the development pattern would be closer to "built-in integrations".

If you agree, could you send a PR to clarify the docs?

Reply:
I'll take a look.  I vaguely remember being surprised that it was needed, but that at least at the time (circa 0.7) it actually was needed to avoid an RBAC error in the kueue manager (perhaps because there were Workloads that were owned by instances of the CRD).